test/hotspot/jtreg/compiler/c2/TestCastX2NotProcessedIGVN.java

```diff
@@ -0,0 +1,78 @@
+package compiler.c2;/*
```

Comment:
Please start the copyright header in a new line.

Reply:
Right! fixed in new commit.